security/login_link.rst

```diff
@@ -804,3 +804,33 @@ features such as the locale used to generate the link::
 
         // ...
     }
+
+
+The ``createLoginLink()`` method accepts a third optional argument to pass an
+``options`` array. There are three keys to consider: route_name, lifetime and parameters.
+$options['route_name'] will override the global route_name value and $options['lifetime'] will override the lifetime global value.
```

Comment:
Should we mention what the global route_name and lifetime values are? Maybe give an example of why might want to override them?